crates/viewer/re_ui/src/color_table.rs

```diff
@@ -0,0 +1,217 @@
+use std::fmt::{Display, Formatter};
+
+/// A hue for a [`ColorToken`].
+#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq)]
+pub enum Hue {
+    Gray,
+    Green,
+    Red,
+    Blue,
+    Purple,
+}
+
+impl nohash_hasher::IsEnabled for Hue {}
+
+impl Hue {
+    pub fn all() -> &'static [Self] {
+        static ALL: [Hue; 5] = [Hue::Gray, Hue::Green, Hue::Red, Hue::Blue, Hue::Purple];
+        &ALL
+    }
+}
+
+impl Display for Hue {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        match self {
+            // these must be as they appear in `design_token.json`
+            Self::Gray => f.write_str("Gray"),
+            Self::Green => f.write_str("Green"),
+            Self::Red => f.write_str("Red"),
+            Self::Blue => f.write_str("Blue"),
+            Self::Purple => f.write_str("Purple"),
+        }
+    }
+}
+
+/// A shade for a [`ColorToken`].
+#[derive(Debug, Clone, Copy, Hash, PartialEq, Eq)]
+pub enum Shade {
```

Comment:
Please document what `Shade` means (at least roughly). Is `S1000` fully saturated, and `S0` gray? Or is there no such inherent meaning?